src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -97,6 +97,27 @@ public function prepare_items() {
 
 		$args['posts_per_page'] = $per_page;
 
+		if ( 'scheduled' === $this->tab->get_slug() && isset( $_REQUEST['s'] ) && ! empty( $_REQUEST['s'] ) ) {
```

Comment:
This will need to be $_REQUEST because the search box form performs the POST, but the pagination links provide the 's' variable within the URL's query string for the GET value.

Reply:
Makes sense :+1: